src/css/components/_captions-settings.scss

```diff
@@ -26,30 +26,36 @@
   .vjs-text-track-settings .vjs-modal-dialog-content {
     display: grid;
     grid-template-columns: 1fr 1fr;
-    grid-template-rows: 1fr auto;
+    grid-template-rows: 1fr;
+    // flex an grid for firefox, ie, edge remove bottom padding/margin in a container as size decreases
```

Comment:
This comment isn't clear to me - is there a typo in "flex an grid"? Is it "... for firefox, ie & edge" or "... for firefox. ie & edge ..."?

Reply:
yeah, should be `flex and grid`